### ISSUE TITLE
Warn about trailing slashes in "webwork.server"

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#866]
See PR #865.  Once WW refactor settles down, we could add a check and warning within the code.

**Comment by rbeezer** [comment]
Even better!  Maybe both: "..., so a trailing slash is being removed"

**Comment by Alex-Jordan** [comment]
I was looking into addressing this, but I think the issue went away with the big webwork refactor. The issue here was reported back when you needed to declare the server as a stringparam when building HTML. The reported issue is that using a trailing slash in the HTML would cause trouble. 

But with the refactor, the only time you declare a server URL is when you do the extraction. The URL is (sort of) sanitized there, with a slash added even if one was missing, and then everything is assuming the appropriate number of slashes. 

So it's not a problem now. Now it's fine to use `https://webwork-ptx.aimath.org` or `https://webwork-ptx.aimath.org/`. _However_ it might be a problem if someone used `https://webwork-ptx.aimath.org//`. I could change the `sanitize_url` subroutine in `mbx` to use tools from `urllib` to _really_ sanitize the input. (Actually, it's only used in one place. I'm not sure there is a real need to modularize it. So I could just get rid of it?) How does that sound?

**Comment by rbeezer** [comment]
You want to get rid of `sanitize_url`, right?  I never am sure what "it" is.  ;-)

If it is only used once, then yes, let's trash it.  It can always rise from the dead via the git history.

**Comment by Alex-Jordan** [comment]
Yes, I mean that I could get rid of `sanitize_url` since I'm not sure it is worth the modularization. Either way, there would be a small chunk of code that does a better job of sanitizing a url. Just maybe more in an inline fashion if `sanitize_url` goes away.

**Comment by rbeezer** [comment]
Sounds good.  Maybe comment that the good chunk could be modularized if it ever 
gets recycled.

**Comment by rbeezer** [comment]
The `webwork.server` string parameter has been superseded by the publication file's `webwork/@server`, and the code now reliably normalizes the server URL, adding a trailing slash when one is not authored. The warning proposed here is therefore unnecessary. Closing as resolved.

*Claude Fable 5, acting as a review assistant for Rob Beezer*
